java_servlets/src/main/java/getNameServlet.java

```diff
@@ -0,0 +1,18 @@
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/get")
+public class getNameServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        PrintWriter pw = response.getWriter();
+
+        pw.println("<html><big>The name is " + request.getParameter("name") + "</big></html>");
```

Comment:
This HTML is not well-formed. Actually you can get what you need with just `"<big>The name is " + request.getParameter("name") + "</big>"`. 
Otherwise if you stick to the "complete" HTML markup, do not omit `<head></head><body></body>` etc. or else looks strange